packages/docusaurus-plugin-content-blog/src/__tests__/frontMatter.test.ts

```diff
@@ -282,6 +282,23 @@ describe('validateBlogPostFrontMatter slug', () => {
   });
 });
 
+describe('validateBlogPostFrontMatter aliases', () => {
+  testField({
+    prefix: 'aliases',
+    validFrontMatters: [
+      {aliases: ['blog/']},
+      {aliases: ['/blog']},
+      {aliases: ['/blog/']},
+      {aliases: ['./blog']},
+      {aliases: ['../blog']},
+      {aliases: ['../../blog']},
+      {aliases: ['/api/plugins/@docusaurus/plugin-debug']},
+      {aliases: ['@site/api/asset/image.png']},
+    ],
+    invalidFrontMatters: [[{aliases: ['']}, 'not allowed to be empty']],
```

Comment:
ensure we don't allow null/undefined/other value arrays and also ensure we don't allow null/undefined as array value (no sparse array allowed)